### ISSUE TITLE
Graphing?

### ISSUE THREAD

**Issue by luckman212** [ndfred/iperf-ios#37]
Thanks @ndfred for making this! It's the only usable iperf3 app in the entire App Store.

I was wondering if you'd considered a simple line graph to visualize the change over time as you walk around and test speeds from different areas?

I looked at every Objective-C charting library listed at [ameizi/awesome-ios-chart](https://github.com/ameizi/awesome-ios-chart/blob/master/Objective-C.md) and selected the top 3 that have been recently updated and have the most stars, for some potential candidates:

[JunyiXie/XJYChart](https://github.com/JunyiXie/XJYChart)
[AAChartModel/AAChartKit](https://github.com/AAChartModel/AAChartKit)
[ZhipingYang/XYChart](https://github.com/ZhipingYang/XYChart)

**Comment by ndfred** [comment]
I would love to get some sort of visualization in, though I am not sure if a graph is the right thing as you don't really care about performance over time so much as some sort of scatter plot of where the bulk of you performance is. I've done some super basic updating mean visualization, but this would be better. If I can get some time to work on it, I'll try and get that in!

On the graphing libraries suggestions, I would like to try and keep the app as small as possible, with as little UI overhead so the test can be super accurate. A simple custom CoreGraphics layer with a callback for every frame is really what I would have in mind, that will then easily be optimized.

**Comment by luckman212** [comment]
@ndfred Thank you for that. I'm not sure I agree about the scatter plot vs. a linear graph. When I use iperf, often I'm walking around to different areas and want to see the effect of changing locations/channels/power levels in realtime. In a scatter plot, you can see the distribution but it's impossible to correlate a point in time with the performance level.

**Comment by ndfred** [comment]
That is true as well, I'll need to play with both options and see. Another concern is the performance overhead of a full graphing solution, as I would assume these are mostly meant to be static rather than dynamic.
